dlt/destinations/impl/bigquery/bigquery.py

```diff
@@ -217,32 +223,53 @@ def restore_file_load(self, file_path: str) -> LoadJob:
         return job
 
     def start_file_load(self, table: TTableSchema, file_path: str, load_id: str) -> LoadJob:
-        job = super().start_file_load(table, file_path, load_id)
```

Comment:
you should keep this first. it will launch various sql jobs ie merge operations. tables that do insert also participate in this